README.markdown

```diff
@@ -186,8 +186,33 @@ instance FiniteBits Foo where
 #endif
 ```
 
-This is a very unsatisfactory solution, and for this reason, we do not pursue
-it. For similar reasons, we do not backport data types.
+Another approach is to backport the API from `base-4.7.0.0` and to declare
+additional methods outside of the class:
+
+```
+#if MIN_VERSION_base(4,7,0) && !(MIN_VERSION_base(4,8,0))
+countLeadingZeros :: FiniteBits b => b -> Int
+countLeadingZeros = {- default implementation #-}
+#endif
+```
+
+The situation is only sligtly better for classes which exist across all `base` version,
```

Comment:
for classes which exist across all versions of `base`

(Also there's a typo: s/sligtly/slightly/)